app/index.php

```diff
@@ -15,32 +27,59 @@
     <title>Book List</title>
 </head>
 <body>
+
 <div class="stickyFooterExcluder">
 
     <?php include "includes/header.php" ?>
 
-        <div class="container">
-            <div class="row">
-                <div class="bookList col-xs-9 col-xs-offset-3">
-                    <?php
-                    if (!$books) {
-                        echo '<div class="alert alert-danger" role="alert">"Something goes wrong, please try again later"</div>';
-                    } else {
-                        foreach ($books as $book) { ?>
-                            <div class='listedBook col-xs-4'>
-                                <a href='individualBookPage.php?id=<?php echo $book->id; ?>'>
-                                    <img class='bookImage' src='<?php echo $book->image; ?>'>
-                                    <h4 class='title'><?php echo $book->title; ?></h4>
-                                    <h4 class='price'><?php echo $book->displayPrice(); ?></h4>
+    <div class="container">
+        <div class="row">
+            <div class="searchAndFilterColumn col-xs-3">
+                <form class="searchForm col-xs-12">
+                    <input class="searchInput col-xs-8" type="text" placeholder="Type here...">
+                    <button class="searchButton btn btn-default col-xs-4" type="button">Search</button>
```

Comment:
This needs to be `type="submit"` rather than `type="button"`, in order to make the form submit when it is clicked. Should be fixed when 4searchTitle is merged in but double check.